config/alpha.json

```diff
@@ -5,7 +5,6 @@
   "hash": "alpha",
   "apiUrl": "https://consent.dsde-alpha.broadinstitute.org",
   "ontologyApiUrl": "https://consent-ontology.dsde-alpha.broadinstitute.org/",
-  "clientId": "1020846292598-hd801vsmmbhh97vaf6aar17lu0q2evfj.apps.googleusercontent.com",
```

Comment:
I thought alpha didn't exist anymore? Why do we need this config?

Reply:
This config file name is not important and is, unfortunately, an artifact of our build image process. Originally, it did reference real environment variables. I'll change this in a separate PR/ticket.

Reply:
I created https://broadworkbench.atlassian.net/browse/DCJ-678 to cover this bit of tech debt.